pkg/webhook/merge_on_review_approval_and_status_checks.go

```diff
@@ -169,5 +169,13 @@ func mergePR(issue *github.Issue, owner, repository string, gh *github.Client, c
 		return errors.Wrapf(err, "failed to merge pull request %s", issue.GetHTMLURL())
 	}
 
+	comment := fmt.Sprintf("Fixes #%d", issue.GetNumber())
```

Comment:
With the github API, a PR is also an issue... so `issue.GetNumber()` is actually the PR number... so this comment is saying the PR fixes itself!